mlir/test/rocmlir-driver/misc_e2e/conv2d_harness_cpu_verifier_kyxc_nhwc_nhwk.mlir

```diff
@@ -11,34 +11,22 @@ module {
     %1 = memref.alloc() : memref<128x32x32x1x8xf32>
     %2 = memref.alloc() : memref<128x30x30x1x128xf32>
 
-    %3 = memref.cast %0 : memref<1x128x3x3x8xf32> to memref<?x?x?x?x?xf32>
-    %4 = memref.cast %1 : memref<128x32x32x1x8xf32> to memref<?x?x?x?x?xf32>
-    %5 = memref.cast %2 : memref<128x30x30x1x128xf32> to memref<?x?x?x?x?xf32>
-
-    // populate initial values.
-    %c0_i16 = arith.constant 0 : i16
-    %c1_i16 = arith.constant 1 : i16
-    %c1_i16_0 = arith.constant 1 : i16
-    %c1_i32 = arith.constant 1 : i32
-    call @mcpuMemset5DFloatRandInt(%3, %c1_i16, %c1_i16_0, %c1_i32) : (memref<?x?x?x?x?xf32>, i16, i16, i32) -> ()
-    call @mcpuMemset5DFloatRandInt(%4, %c1_i16, %c1_i16_0, %c1_i32) : (memref<?x?x?x?x?xf32>, i16, i16, i32) -> ()
-    call @mcpuMemset5DFloatRandInt(%5, %c0_i16, %c0_i16, %c1_i32) : (memref<?x?x?x?x?xf32>, i16, i16, i32) -> ()
+    linalg.fill ins(%cst : f32) outs(%0 : memref<1x128x3x3x8xf32>)
```

Comment:
Does it fill in random numbers?

Reply:
No, that's meant to be a fixed value. Note that the old calls to "Rand" had the same min and max value